src/ROUTES.ts

```diff
@@ -100,7 +100,13 @@ export default {
         getRoute: (contactMethod: string) => `settings/profile/contact-methods/${encodeURIComponent(contactMethod)}/details`,
     },
     SETTINGS_NEW_CONTACT_METHOD: 'settings/profile/contact-methods/new',
-    SETTINGS_2FA: 'settings/security/two-factor-auth',
+    SETTINGS_2FA: {
+        CODES: 'settings/security/two-factor-auth/codes',
+        VERIFY: 'settings/security/two-factor-auth/verify',
+        SUCESS: 'settings/security/two-factor-auth/success',
+        ENABLED: 'settings/security/two-factor-auth/enabled',
+        DISABLED: 'settings/security/two-factor-auth/disabled',
```

Comment:
Did you check how deeplinking is performed with new routes? Where are we prohibiting accessing the `SUCCESS` route directly via deeplink?

Reply:
I'll add code from https://github.com/Expensify/App/pull/28895#discussion_r1356711352 to fix this.